guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md

```diff
@@ -29,10 +39,10 @@ The sample configuration overrides your server's docroot settings to serve files
               set $MAGE_ROOT /var/www/html/magento2ce;
               include /var/www/html/magento2ce/nginx.conf.sample;
     }
-
+```
```

Comment:
Just make sure there is an empty line between the code ticks and Before you begin section.

Reply:
THanks.